<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-AL3VMHN" name="com.nopcommerce.admin.Admin" tests="32" failures="4" timestamp="17 Sep 2021 07:07:25 GMT" time="272.159" errors="0">
  <testcase name="Admin_00_Login_To_Admin_Page" time="7.800" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_02_Search_With_Product_Name_Parent_Category_Unchecked" time="2.949" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_01_Search_With_Product_Name" time="3.596" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_03_Search_With_Product_Name_Parent_Category_Checked" time="5.168" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_11_Search_Customer_With_Full_Data" time="6.518" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_06_Go_Directly_To_SKU" time="6.249" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_04_Search_With_Product_Name_Child_Category" time="4.616" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_13_Add_New_Address" time="17.487" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_02_Search_With_Product_Name_Parent_Category_Unchecked" time="4.070" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_08_Search_Customer_With_Email" time="4.704" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_10_Search_Customer_With_Company" time="6.910" classname="com.nopcommerce.admin.Admin">
    <failure type="java.lang.AssertionError" message="expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [2]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at commons.BaseTest.checkEquals(BaseTest.java:182)
at commons.BaseTest.verifyEquals(BaseTest.java:194)
at com.nopcommerce.admin.Admin.Admin_10_Search_Customer_With_Company(Admin.java:309)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Admin_10_Search_Customer_With_Company -->
  <testcase name="Admin_01_Search_With_Product_Name" time="3.845" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_07_Create_New_Customer" time="15.770" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_15_Delete_Address" time="9.416" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_12_Edit_Customer" time="18.599" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_00_Login_To_Admin_Page" time="3.304" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_11_Search_Customer_With_Full_Data" time="5.362" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_10_Search_Customer_With_Company" time="6.684" classname="com.nopcommerce.admin.Admin">
    <failure type="java.lang.AssertionError" message="expected [1] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [3]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at commons.BaseTest.checkEquals(BaseTest.java:182)
at commons.BaseTest.verifyEquals(BaseTest.java:194)
at com.nopcommerce.admin.Admin.Admin_10_Search_Customer_With_Company(Admin.java:309)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Admin_10_Search_Customer_With_Company -->
  <testcase name="Admin_07_Create_New_Customer" time="18.968" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_09_Search_Customer_With_FirstName_LastName" time="6.842" classname="com.nopcommerce.admin.Admin">
    <failure type="java.lang.AssertionError" message="expected [1] but found [3]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [3]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at commons.BaseTest.checkEquals(BaseTest.java:182)
at commons.BaseTest.verifyEquals(BaseTest.java:194)
at com.nopcommerce.admin.Admin.Admin_09_Search_Customer_With_FirstName_LastName(Admin.java:296)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Admin_09_Search_Customer_With_FirstName_LastName -->
  <testcase name="Admin_05_Search_With_Product_Name_Manufacturer" time="3.743" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_12_Edit_Customer" time="15.729" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_14_Edit_Address" time="18.403" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_06_Go_Directly_To_SKU" time="6.018" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_09_Search_Customer_With_FirstName_LastName" time="7.670" classname="com.nopcommerce.admin.Admin">
    <failure type="java.lang.AssertionError" message="expected [1] but found [2]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [2]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at commons.BaseTest.checkEquals(BaseTest.java:182)
at commons.BaseTest.verifyEquals(BaseTest.java:194)
at com.nopcommerce.admin.Admin.Admin_09_Search_Customer_With_FirstName_LastName(Admin.java:296)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Admin_09_Search_Customer_With_FirstName_LastName -->
  <testcase name="Admin_03_Search_With_Product_Name_Parent_Category_Checked" time="3.797" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_05_Search_With_Product_Name_Manufacturer" time="4.264" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_14_Edit_Address" time="17.336" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_08_Search_Customer_With_Email" time="4.878" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_04_Search_With_Product_Name_Child_Category" time="5.734" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_15_Delete_Address" time="8.725" classname="com.nopcommerce.admin.Admin"/>
  <testcase name="Admin_13_Add_New_Address" time="17.005" classname="com.nopcommerce.admin.Admin"/>
</testsuite> <!-- com.nopcommerce.admin.Admin -->
